server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/dto/DLPConfigurationItemDTO.java

```diff
@@ -0,0 +1,140 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.webadmin.dto;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.apache.james.dlp.api.DLPConfigurationItem;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.steveash.guavate.Guavate;
+import com.google.common.base.Preconditions;
+
+public class DLPConfigurationItemDTO {
+
+    public static List<DLPConfigurationItem> toDLPConfigurations(List<DLPConfigurationItemDTO> dtos) {
+        Preconditions.checkNotNull(dtos);
+
+        return dtos.stream()
+            .map(DLPConfigurationItemDTO::toDLPConfiguration)
+            .collect(Guavate.toImmutableList());
+    }
+
+    public static List<DLPConfigurationItemDTO> toDTOs(List<DLPConfigurationItem> dlpConfigurations) {
+        Preconditions.checkNotNull(dlpConfigurations);
+
+        return dlpConfigurations.stream()
+            .map(DLPConfigurationItemDTO::toDTO)
+            .collect(Guavate.toImmutableList());
+    }
+
+    public static DLPConfigurationItem toDLPConfiguration(DLPConfigurationItemDTO dto) {
+        return DLPConfigurationItem.builder()
+            .id(DLPConfigurationItem.Id.of(dto.id))
+            .expression(dto.expression)
+            .explanation(dto.explanation)
+            .targetsSender(dto.targetsSender)
+            .targetsRecipients(dto.targetsRecipients)
+            .targetsContent(dto.targetsContent)
+            .build();
+    }
+
+    public static DLPConfigurationItemDTO toDTO(DLPConfigurationItem dlpConfiguration) {
```

Comment:
do you need it to be public ?

Reply:
I need to use that method on `DLPConfigurationItemDTOTest` 😆

Reply:
Then it can be `@VisibleForTesting`